patches/@edx+frontend-build+13.0.14.patch

```diff
@@ -0,0 +1,100 @@
+diff --git a/node_modules/@edx/frontend-build/config/jest.config.js b/node_modules/@edx/frontend-build/config/jest.config.js
+index ae08b70..2c8c0fa 100644
+--- a/node_modules/@edx/frontend-build/config/jest.config.js
++++ b/node_modules/@edx/frontend-build/config/jest.config.js
+@@ -3,12 +3,16 @@ const fs = require('fs');
+ 
+ const presets = require('../lib/presets');
+ 
++console.log('If you are expecting to use an env.config.jsx config, make sure to run `npm run postinstall` first!')
```

Comment:
[clarification] I'm assuming this console.log wouldnt be in the upstream contribution to `@edx/frontend-build`?

Reply:
Yes to it not being added upstream although... I'm now realizing that the `postinstall` script should be able to handle this when anyone runs `npm install`. I'll remove it!